.circleci/continue_config.yml

```diff
@@ -462,6 +462,8 @@ workflows:
           docker_name: op-conductor-mon
           docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
           docker_context: .
+          requires:
```

Comment:
is this supposed to be part of this change?

Reply:
Prior to this change when the branch was up for PR, it was executing the op-conductor-mon release worklow and failing. 

This changed stoped that from executing, I'm happy to open up this smaller update within a new PR

Reply:
no needed, just want to confirm/clarify